pyfar/dsp/dsp.py

```diff
@@ -2137,11 +2136,11 @@ def normalize(signal, reference_method='max', domain='time',
 
     if domain not in ('time', 'freq'):
         raise ValueError("domain must be 'time' or 'freq'.")
-    if type(signal) == pyfar.FrequencyData and domain == 'time':
+    if isinstance(signal, pyfar.FrequencyData) and domain == 'time':
```

Comment:
I think this does not work, because pyfar Signals are instance of Time and Frequency data due to the inheritance
```
import pyfar as pf

signal = pf.Signal([1, 2], 44100)

assert isinstance(signal, pf.TimeData)
assert isinstance(signal, pf.FrequencyData)
```

Reply:
I fixed all of them, I thought i have done it before somewhere...